src/lms-harmonizer/edfi_lms_harmonizer/helpers/constants.py

```diff
@@ -4,3 +4,15 @@
 # See the LICENSE and NOTICES files in the project root for more information.
 
 LOG_LEVELS = ["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"]
+
+
+class SOURCE_SYSTEM:
+    GOOGLE = "Google"
+    SCHOOLOGY = "Schoology"
+    CANVAS = "Canvas"
+
+
+class SOURCE_SYSTEM_NAMESPACE:
+    GOOGLE = "uri://google.com/classroom"
+    SCHOOLOGY = "uri://schoology.com"
+    CANVAS = "uri://instructure.com/canvas"
```

Comment:
We need to make sure that we document these namespace requirements for any API clients. Perhaps we can do that in LMS-298 "Define security requirements". I'll make a note in that ticket.